### PR TITLE
make properties configurable in ip_informer rule

### PR DESCRIPTION
**Pull request by ekneg54**

### CODE REVIEW

File: tests/unit/processor/ip_informer/test_ip_informer.py [hunk 1]
```diff
@@ -183,6 +184,46 @@
...
+                "target_field": "result",
+                "properties": ["is_loopback"],
+            },
+        },
+        {"ip": "192.168.5.1"},
+        {
+            "ip": "192.168.5.1",
+            "result": {
+                "192.168.5.1": {
+                    "is_loopback": False,
```
Comment (urdit52) [inline tests/unit/processor/ip_informer/test_ip_informer.py:202]: would it make more sense to test for a property that is != False in order to differentiate from the second test case?

**Review by urdit52 — CHANGES_REQUESTED** [review]
2 remarks.

File: tests/unit/processor/ip_informer/test_ip_informer.py (hunk 1 above)
Comment (ekneg54) [reply tests/unit/processor/ip_informer/test_ip_informer.py:202]: this testcase tests that it should be possible to only extract a single property. 
the second testscase tests, that it should be possible to extract a properte which does not exist for a ipv4address.
the value of the fields does not matter for the first testcase.

Comment (urdit52) [reply tests/unit/processor/ip_informer/test_ip_informer.py:202]: fair enough.

**Review by urdit52 — APPROVED** [review]
thank you for the clarification and modification. looks good to me.

**Pull request merged**